routines/epochtal.js

```diff
@@ -263,7 +263,7 @@ async function releaseMap (context) {
       fs.rmSync(portal2.output, { recursive: true });
     }
 
-    fs.renameSync(vmf, `${__dirname}/../vmfs/${context.data.week.map.id}.vmf.xz`);
+    fs.renameSync(vmf, `${context.file.vmfs}/${context.data.week.map.id}.vmf`);
```

Comment:
The `compress` argument for the `gamefiles/getvmf` util call is `true`, so this should be a `.vmf.xz`